pkg/sbom/nodelist.go

```diff
@@ -21,6 +21,14 @@ type edgeIndex map[string]map[Edge_Type][]*Edge
 // rootElementsIndex is an index of the top levele elements by ID
 type rootElementsIndex map[string]struct{}
 
+// hashIndex is a struct that indexes a node list by has values
```

Comment:
What is "by has values?"

Reply:
Lol fixed it to "by the hash values of its nodes"